From 305b732ef692000f47b02fece94b037bdf75a676 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Mon, 28 Nov 2011 23:42:15 +0900
Subject: [PATCH 5/5] copyright-format: English proofreading by
 debian-l10n-english.

Wording: Justin B Rye <jbr@edlug.org.uk>
Seconded: Charles Plessy <plessy@debian.org>
Closes: #648387
---
 copyright-format/copyright-format.xml |   80 ++++++++++++++++----------------
 1 files changed, 40 insertions(+), 40 deletions(-)

diff --git a/copyright-format/copyright-format.xml b/copyright-format/copyright-format.xml
index 43194e5..90cab96 100644
--- a/copyright-format/copyright-format.xml
+++ b/copyright-format/copyright-format.xml
@@ -25,10 +25,10 @@
     </legalnotice>
     <abstract>
       <para>
-        This specification was drafted as <ulink
+        This specification was drawn up as <ulink
         url="http://dep.debian.net/deps/dep5/">DEP-5</ulink>, to establish a
         standard, machine-readable format for
-        <filename>debian/copyright</filename> files within packages and
+        <filename>debian/copyright</filename> files within packages and to
         facilitate automated checking and reporting of licenses for packages and
         sets of packages.
       </para>
@@ -81,9 +81,9 @@
       no way to know how much of Debian should be stripped from such a system.
     </para>
     <para>
-      A user might want to have a way to avoid software with certain licenses
-      they have a problem with, even if the licenses are DFSG-free. For example,
-      the Affero GPL.
+      Even where licenses are DFSG-free and mutually compatible, users might want
+      a way to avoid software with certain licenses, for example if they have a
+      problem with the Affero GPL.
     </para>
   </section>
 
@@ -121,32 +121,32 @@
       avoid conflicting specifications for widely used extra fields.
     </para>
     <para>
-      There are four kinds values for fields.  Each field specifies which kind
-      is allowed.
+      Each field must have one of the following four types of value,
+      as specified in the field's definition.
     </para>
 
     <section id="single-line">
       <title>Single-line values</title>
       <para>
-        A single-line value means that the whole value of a field must fit on a
+        This means that the whole value of a field must fit on a
         single line.  For example, the <varname>Format</varname> field has a
-        single line value specifying the version of the machine-readable format
+        single-line value specifying the version of the machine-readable format
         that is used.
       </para>
     </section>
 
     <section id="white-space-lists">
-      <title>White space separated lists</title>
+      <title>Whitespace-separated lists</title>
       <para>
-        A white space separated list means that the field value may be on one
-        line or many, but values in the list are separated by one or more white
-        space characters (including space, TAB, and newline).  For example, the
+        This means that the field value may be on one
+        line or many, but values in the list are separated by one or more
+        whitespace characters (including space, TAB, and newline).  For example, the
         <varname>Files</varname> field has a list of filename patterns.
       </para>
     </section>
 
     <section id="line-based-lists">
-      <title>Line based lists</title>
+      <title>Line-based lists</title>
       <para>
         Another kind of list value has one value per line. For example,
         <varname>Copyright</varname> can list many copyright statements, one per
@@ -159,7 +159,7 @@
       <para>
         Formatted text fields use the same rules as the long description in a
         package's <varname>Description</varname> field, possibly also using the
-        first line  as a synopsis, like <varname>Description</varname> uses it
+        first line  as a synopsis, just as <varname>Description</varname> uses it
         for the short description. See Debian Policy's section 5.6.13, <ulink
         url="http://www.debian.org/doc/debian-policy/ch-controlfields#s-f-Description"><quote>Description</quote></ulink>,
         for details.  For example, <varname>Disclaimer</varname> has no special
@@ -174,17 +174,17 @@
       There are three kinds of paragraphs: the first one is called the
       <link linkend="header-paragraph">header paragraph</link>.  Every other
       paragraph is either a <link linkend="files-paragraph">Files</link>
-      paragraph or a <link linkend="stand-alone-license-paragraph">stand-alone
+      paragraph or a <link linkend="stand-alone-license-paragraph">standalone
       license</link> paragraph.  This is similar to source and binary package
       paragraphs in <filename>debian/control</filename> files.
     </para>
 
     <section id="header-paragraph">
-      <title>Header paragraph (Once)</title>
+      <title>Header paragraph (once)</title>
       <section id="format-header-field">
         <title><varname>Format</varname></title>
         <para>
-          Required single line: URI of the format specification, such as:
+          Required single line: URI of the format specification, such as
           <literal>http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/</literal>
         </para>
       </section>
@@ -199,7 +199,7 @@
       <section id="upstream-contact-header-field">
         <title><varname>Upstream-Contact</varname></title>
         <para>
-          Optional line based list: the preferred address(es) to reach the
+          Optional line-based list: the preferred address(es) to reach the
           upstream project.  May be free-form text, but by convention  will
           usually be written as a list of RFC5322 addresses or URIs.
         </para>
@@ -208,7 +208,7 @@
       <section id="source-header-field">
         <title><varname>Source</varname></title>
         <para>
-          Optional formatted text, no synopsis: an explanation from where the
+          Optional formatted text, no synopsis: an explanation of where the
           upstream source came from. Typically this would be a URL, but it might
           be a free-form explanation.  The Debian Policy section <ulink
           url="http://www.debian.org/doc/debian-policy/ch-docs#s-copyrightfile">12.5</ulink>
@@ -247,7 +247,7 @@
           (no <varname>Files</varname> specification), this field gives the
           license information for the package as a whole, which may be different
           or simplified from a combination of all the per-file license
-          information. <varname>License</varname> below in the <link
+          information. See also <varname>License</varname> below in the <link
           linkend="files-paragraph">Files paragraph</link> section.
         </para>
       </section>
@@ -255,7 +255,7 @@
       <section id="copyright-header-field">
         <title><varname>Copyright</varname></title>
         <para>
-          Optional line based list: in the header paragraph (no
+          Optional line-based list: in the header paragraph (no
           <varname>Files</varname> specification), this field gives the
           copyright information for the package as a whole, which may be
           different or simplified from a combination of all the per-file
@@ -286,9 +286,9 @@ Source: http://www.example.com/software/project</programlisting>
     </section>
 
     <section id="files-paragraph">
-      <title>Files paragraph (Repeatable)</title>
+      <title>Files paragraph (repeatable)</title>
       <para>
-        The declaration of copyright and license for files is done in one or
+        The declaration of copyright and license for files may consist of one or
         more paragraphs.  In the simplest case, a single paragraph can be used
         which applies to all files and lists all applicable copyrights and
         licenses.
@@ -297,7 +297,7 @@ Source: http://www.example.com/software/project</programlisting>
       <section id="files-files-field">
         <title><varname>Files</varname></title>
         <para>
-          Required white space separated list: list of patterns indicating files
+          Required whitespace-separated list: list of patterns indicating files
           covered by the license and copyright specified in this paragraph.
         </para>
         <para>
@@ -360,15 +360,15 @@ Source: http://www.example.com/software/project</programlisting>
         paragraph that matches a particular file applies to it.
       </para>
       <para>
-        Exclusions are done by having multiple <varname>Files</varname>
-        paragraphs.
+        Exclusions are only supported by adding <varname>Files</varname>
+        paragraphs to override the previous match.
       </para>
       </section>
 
       <section id="copyright-files-field">
         <title><varname>Copyright</varname></title>
         <para>
-          Required line based list: one or more free-form copyright
+          Required line-based list: one or more free-form copyright
           statement(s), one per line, that apply to the files matched by the
           above pattern.  If a work has no copyright holder (i.e., it is in
           the public domain), that information should be recorded here.
@@ -413,8 +413,8 @@ Copyright 2009, 2010 Angela Watts</programlisting>
           copyright file.
         </para>
         <para>
-          Remaining lines: if left blank here, the file
-          <emphasis>must</emphasis> include a stand-alone
+          Remaining lines: if these are omitted, the file
+          <emphasis>must</emphasis> include a standalone
           <varname>License</varname> paragraph matching each license short
           name listed on the first line (see the <link
           linkend="stand-alone-license-paragraph">Standalone License
@@ -467,10 +467,10 @@ License: GPL-2+</programlisting>
     </section>
 
     <section id="stand-alone-license-paragraph">
-      <title>Standalone License Paragraph (Optional, Repeatable)</title>
+      <title>Standalone license paragraph (optional, repeatable)</title>
       <para>
-        Where a set of files are dual (tri, etc) licensed, or when the same
-        license occurs multiple times, you can use a single line
+        Where a set of files are covered by multiple licenses, or one
+        license occurs multiple times, you can use a single-line
         <varname>License</varname> field and standalone
         <varname>License</varname> paragraphs to expand the license short names.
       </para>
@@ -541,7 +541,7 @@ License: MPL-1.1
         <filename>debian/copyright</filename>, nor any requirements in the
         license of the work regarding reproduction of legal notices.  This
         information must still be included in the <varname>License</varname>
-        field, either in a stand-alone license paragraph or in the relevant
+        field, either in a standalone license paragraph or in the relevant
         files paragraph.
       </para>
       <para>
@@ -549,9 +549,9 @@ License: MPL-1.1
         added, using a dash as a separator. If omitted, the lowest version
         number is implied. When the license grant permits using the terms of any
         later version of that license, the short name is finished with a plus
-        sign. For <link linkend="spdx">SPDX</link> compatibility, trailing
-        <emphasis>dot-zeroes</emphasis> are considered to be equal to plainer
-        version (e.g., <quote>2.0.0</quote> is considered equal to
+        sign. For <link linkend="spdx">SPDX</link> compatibility, versions with trailing
+        <emphasis>dot-zeroes</emphasis> are considered to be equivalent to
+        versions without (e.g., <quote>2.0.0</quote> is considered equal to
         <quote>2.0</quote> and <quote>2</quote>).
       </para>
       <para>
@@ -884,7 +884,7 @@ License: MPL-1.1
         matches.
       </para>
       <para>
-        An exception or clarification to a license is signaled in plain text, by appending
+        An exception or clarification to a license is signalled in plain text, by appending
         <literal>with <varname><replaceable>keywords</replaceable></varname>
         exception</literal> to the short name.  This document provides a list of
         keywords that must be used when referring to the most frequent
@@ -921,7 +921,7 @@ statement from your version.</programlisting>
           The GPL <literal>OpenSSL</literal> exception gives permission to link GPL-licensed
 code with the OpenSSL library, which contains GPL-incompatible clauses.
 For more information, see <ulink
-url="http://www.gnome.org/~markmc/openssl-and-the-gpl">The -OpenSSL License and
+url="http://www.gnome.org/~markmc/openssl-and-the-gpl">The OpenSSL License and
 The GPL</ulink> by Mark
 McLoughlin and the message <ulink
 url="http://lists.debian.org/debian-legal/2004/05/msg00595.html">middleman
@@ -992,7 +992,7 @@ also delete it here.</programlisting>
         This is for a file that has both GPL and classic BSD code in it:
 <programlisting>License: GPL-2+ and BSD</programlisting>
         For the most complex cases, the comma is used to disambiguate the
-        priority of <literal>or</literal>s and <literal>and</literal>s
+        priority of <literal>or</literal>s and <literal>and</literal>s;
         <literal>and</literal> has the priority over <literal>or</literal>,
         unless preceded by a comma. For instance:
       </para>
-- 
1.7.5.4

